ffi/diplomat/cpp/examples/pluralrules/Makefile

```diff
@@ -16,7 +16,7 @@ $(ALL_HEADERS):
 
 
 ../../../../../target/debug/libicu_capi_staticlib_test.a: $(ALL_RUST)
-	cargo build -p icu_capi_staticlib --features cpp_default,provider_fs,icu_provider/deserialize_json
+	cargo build -p icu_capi_staticlib -p icu_provider --features cpp_default,provider_fs,deserialize_json
```

Comment:
TIL you can do that

what happens if the feature names clash, though

Reply:
They'll both be activated. We do this in the WASM CI, something like `cargo build --workspace --features serde`.

Reply:
Ah, fair.


I recommend checking out the tutorials, btw, they may need changes to CLI invocations